### PR TITLE
stm32: clock_control: Kconfig to dts leftovers

### PR DESCRIPTION
**Pull request by erwango**
Hopefully the last bits

### CODE REVIEW

**Review by gmarull — APPROVED** [review]

File: boards/arm/nucleo_wb55rg/nucleo_wb55rg.dts [hunk 1]
```diff
@@ -88,6 +88,10 @@
 	status = "okay";
 };
 
+&clk_lse{
```
Comment (FRASTM) [inline boards/arm/nucleo_wb55rg/nucleo_wb55rg.dts:91]: `&clk_lse {`

**Review by FRASTM — COMMENTED** [review]
else LGTM

File: tests/drivers/watchdog/wdt_basic_api/boards/disco_l475_iot1.overlay [hunk 2]
```diff
@@ -1,3 +1,8 @@
+&rrc {
```
Comment (tagunil) [inline tests/drivers/watchdog/wdt_basic_api/boards/disco_l475_iot1.overlay:1]: &rcc?

File: tests/drivers/watchdog/wdt_basic_api/boards/nucleo_f091rc.overlay [hunk 3]
```diff
@@ -4,6 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+&rrc {
```
Comment (tagunil) [inline tests/drivers/watchdog/wdt_basic_api/boards/nucleo_f091rc.overlay:7]: same here

File: tests/drivers/watchdog/wdt_basic_api/boards/nucleo_g431rb.overlay [hunk 4]
```diff
@@ -4,6 +4,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+&rrc {
```
Comment (tagunil) [inline tests/drivers/watchdog/wdt_basic_api/boards/nucleo_g431rb.overlay:7]: and here

**Review by galak — APPROVED** [review]

File: tests/drivers/watchdog/wdt_basic_api/boards/disco_l475_iot1.overlay (hunk 2 above)
Comment (erwango) [reply tests/drivers/watchdog/wdt_basic_api/boards/disco_l475_iot1.overlay]: Yes, fixed, txs!

**Pull request merged**